core/src/error.rs

```diff
@@ -493,6 +496,10 @@ pub enum ParseError {
         /// The previous instance of the duplicated identifier.
         prev_ident: LocIdent,
     },
+    DisabledFeature {
```

Comment:
Even if short, a description comment would be nice, in line with other's variant.